src/components/CarSearch.tsx

```diff
@@ -83,7 +83,8 @@ class CarSearch extends React.Component<calls, {}> {
                               {/* TODO: Use the respective car Image */}
                               <img
                                 className="pl-2 rounded-full"
-                                src="https://upload.wikimedia.org/wikipedia/commons/c/cc/Car_Icon.svg"
+                                src={car.image}
+                                alt=""
```

Comment:
Maybe we could set the `alt` to something descriptive like `{\`${car.make} ${car.model} ${car.year}\`}`?